src/core/opamStd.ml

```diff
@@ -995,19 +995,27 @@ module OpamSys = struct
       (fun f -> try f () with _ -> ())
       !registered_at_exit
 
-  let is_cygwin_variant =
+  let get_windows_executable_variant =
     if Sys.win32 then
       let results = Hashtbl.create 17 in
-      let requires_cygwin name =
+      let requires_cygwin_or_msys2 name =
```

Comment:
The renaming of this function doesn't add anything - please could it go back to `requires_cygwin`?